libyul/backends/evm/EVMCodeTransform.cpp

```diff
@@ -470,16 +510,19 @@ void CodeTransform::operator()(FunctionDefinition const& _function)
 		// But we would like it to be:
 		// <return values...> <return label>?
 		// So we have to append some SWAP and POP instructions.
+		// Note that <arguments...> is not the full list of function arguments, since some arguments may already have
```

Comment:
Maybe this whole comment is rather redundant in that case? "There can be arguments, but maybe not. Oh and the return values can also be somewhere else".

What is not contiguous about them? Can there be local variables in between?

Reply:
In the first commit it was a bit weird... it may not really have worked there anyways. In the second it's clearer, there return variables should again be contiguous anyways, but I think I haven't adjusted the comment yet.

Not sure if stating that ``<arguments...>`` in a function acting on ``FunctionDefinition`` is not really the full list of arguments to the function - but I'm happy to remove all of it, it was mainly meant to explain the redesign of the shuffling below.